src/components/primitive/button/button.component.styles.ts

```diff
@@ -1,27 +1,58 @@
 import styled from 'styled-components';
-import { StyledButtonProps } from './button.component.props';
-import {
-  handleBorderColors,
-  handleActiveColors,
-  handleDefaultBackground,
-  handleDefaultButtonText,
-} from './button.component.utils';
+import { Box } from '../box/box.component';
+import { Text } from '../text/text.component';
 
-export const StyledButton = styled.button<StyledButtonProps>`
-  padding: 1rem 2rem;
-  border-radius: 4px;
+/**
+ * Styles for Primary Button.
+ */
+export const Primary = styled(Box)`
+  height: 5rem;
+  width: 14.6rem;
```

Comment:
Width should not be hard-coded. Only minWidth will be there.

Reply:
Using relevant padding is always a better option rather than hardcoding dimensions.